lib/job-iteration/iteration.rb

```diff
@@ -185,15 +185,15 @@ def iterate_with_enumerator(enumerator, arguments)
       found_record = false
       @needs_reenqueue = false
 
-      enumerator.each do |object_from_enumerator, index|
+      enumerator.each do |object_from_enumerator, cursor_position|
         # Deferred until 2.0.0
-        # assert_valid_cursor!(index)
+        # assert_valid_cursor!(cursor_position)
 
-        ActiveSupport::Notifications.instrument("each_iteration.iteration", {}) do |tags|
+        tags = { job_class: self.class.name, cursor_position: cursor_position }
```

Comment:
Calling `instrumentation_tags` here is useful if we want to add more default tags in the future.
```suggestion
        tags = instrumentation_tags.merge(cursor_position: cursor_position)
```